Makefile

```diff
@@ -142,6 +145,6 @@ test-e2e-1.26: container-build-single-platform install-tools
 test-e2e-1.25: container-build-single-platform install-tools
```

Comment:
Go ahead and remove the 1.25 E2E test, we only run n-3 version E2E tests since they take a decent amount of time

Reply:
removed